config/sync/core.entity_view_display.rdf_entity.collection.about.yml

```diff
@@ -139,17 +140,15 @@ content:
     type: text_default
     region: content
   field_ar_moderation:
+    type: boolean
     weight: 2
-    label: hidden
+    region: content
+    label: above
     settings:
       format: default
-      format_custom_true: ''
       format_custom_false: ''
-    third_party_settings:
-      template_suggestion:
-        template_suggestion: fieldset
-    type: boolean
-    region: content
+      format_custom_true: ''
+    third_party_settings: {  }
```

Comment:
These changes are causing a regression in the about page:

![moderation-label](https://user-images.githubusercontent.com/442924/93776941-d3c6c080-fc2c-11ea-8d3c-e97f105a8038.png)

The label is now shown, but this duplicates the label that is already there.